.gitignore

```diff
@@ -49,5 +48,7 @@ buck-out/
 android/app/libs
 android/keystores/debug.keystore
 
-# Build
-dist/
```

Comment:
why you remove the `dist` folder from git ignore?

Reply:
Publishing `dist` let anyone install from github commits and not depend on NPM registry only. Very convenient to use unreleased/experimental code.
Os course, is a opinionated way of development.

Please, feel free to remove this part I you are not comfortable with it.

Reply:
I agree it can be super convenient to install directly from github. My concern with the dist folder is that people may forget to build before they commit. So, I would only recommend this if we have scripts configured to build as part of the commit process. To be truly responsible, we should add a test in CircleCI to verify that the dist version correctly matches src. Otherwise I'm afraid the dist folder would get out of sync.

Reply:
I don't know for sure how this check in CircleCI would be. If it's complex is better remove `dist` completely.
At the end anyone can fork and commit ther own stuff. That's why git and github are so great.